RadarSDK/RadarSettings.m

```diff
@@ -43,6 +45,23 @@ + (NSString *)installId {
     return installId;
 }
 
++ (NSString *)sessionId {
+    return [NSString stringWithFormat:@"%.f", [[NSUserDefaults standardUserDefaults] doubleForKey:kSessionId]];
+}
+
++ (BOOL)updateSessionId {
+    double timestampSeconds = [[NSDate date] timeIntervalSince1970];
+    double sessionIdSeconds = [[NSUserDefaults standardUserDefaults] doubleForKey:kSessionId];
```

Comment:
I kind of forget. If the key is not existent, will `sessionIdSeconds` be nil or 0?

Reply:
0